docs/open-source-project/release-notes/tizen-5-0-m2.md

```diff
@@ -0,0 +1,476 @@
+  - Support for some tags of HLS v20 has been added.
+  - The tizen allocator is applied to mediacodec.
+  - The tizen allocator and buffer pool have been applied to the following modules:
+    - Player, MediaCodec, Video360, and VideoConverter
+    - TM1, TM2, TW2, and ARTIK Codec Plugins
+    - Tizenwlsink
+- Camera
+  - Removed deprecated enums related with sound policy.
+  - Added new preview format for depth camera.
+  - Updated preview data structure for RGB format.
+- Media Controller 
+  - Added support for content type and age rating for the playing contents.
+  - Added support for icon setting for controller servers.
+  - Added support for capabilities of controller servers.
+  - Added support for search command.
+- Recorder
+  - Removed deprecated enums related with sound policy.
+- Sound Manager
+  - New API set for USB audio output device is added.
+- Pulseaudio
+  - Added infrastructure of support for airplay (raop) device playback functionality.
+- OpenCV
+  - Version 2.4.9 has been upgraded to 3.4.1.
+  - Deep Neural Network (DNN) module has been enabled. 
+
+
+## Network and Connectivity
+
+### New and changed features
+
+- Network Monitoring
+  - Added intelligent-network-monitoring API set.
+    - Added support for monitoring default connection capability.
+    - Added support for getting wireless link properties.
+    - Added support for watching network interface availability.
+    - Added support for providing various network status information for diagnosis.
+-  Wi-Fi Native CAPI
+  - Added a wifi-manager API for forgetting an access point (AP) asynchronously.
+  - Added a new Wi-Fi security type (WPA_FT_PSK).
+  - Added new config API set for EAP private key and password.
+  - Added new config API set for Hidden AP connection with static IP setting.
+- Network Firewall
+  - Added support for netfilter logging functions.
+  - Added support for rule inserting function to top.
+  - Added a STC API for setting firewall log rule.
+  - Added a STC API for setting firewall netfilter log rule.
+- STC
+  - Added support for tethering data usage monitoring per client.
+  - Added support for tethering data usage limitation per client.
+- Telephony
+  - Optimized D-Bus rules for reducing D-Bus policy scanning time.
+- Wi-Fi SON
+  - Added Wi-Fi SON (Self-organizing Networks) API set.
+    - Added support for managing Wi-Fi Mesh network.
+    - Added support for getting Wi-Fi Mesh network parameters.
+    - Added support for SoftAP to extend existing network coverage.
+      - Added 802.11w Protected Management Frame (PMF) feature.
+      - Added 802.11k Radio Resource Management (RRM) feature.
+- Multi Device Group
+  - Added Multi Device Group API set based on IoTivity.
+    - Added group management support for devices.
+    - Added D2D communication support in a group.
+- Bluetooth
+  - Added BT 5.0 API set for getting 2M Phy support.
+    - Added feature for BLE 5.0.
+      - http://tizen.org/feature/network.bluetooth.le.5_0

```

Comment:
As this is a feature, please use code tag:
`http://tizen.org/feature/network.bluetooth.le.5_0`

Reply:
will accept